builder.sh

```diff
@@ -701,7 +701,7 @@ function extract_machine_build() {
         done
 
         for i in "${remove[@]}"; do
-            unset "${BUILD_MACHINE["$i"]}"
+            unset $BUILD_MACHINE["$i"]
```

Comment:
Shouldn't it be this?

```suggestion
            unset "${BUILD_MACHINE[$i]}"
```

That should pass shellcheck

Reply:
While that would indeed make it pass, I think this should be:
```suggestion
            unset "BUILD_MACHINE[$i]"
```

Reply:
Oh right, it's unsetting! Nicely caught @ludeeus 👍